handbook/engineering/leadership/index.md

```diff
@@ -69,6 +69,18 @@ Our long term vision is to build an organization that attracts and retains the b
 - Owner: TBD.
 - Status: Not started. Needs more discussion.
 
+## Communication
+
+We have a 1hr sync with product and engineering every Tuesday at 9am PT ([meeting notes](https://docs.google.com/document/d/10fEh3Cw0ENKjFYDP-4OoLZbOZTVAtMpifc2WM_1mGVU/edit)).
+
+Engineering managers send an update to engineering-leadership-status@sourcegraph.com by 10am PT every Monday (sending it earlier, like EOD Friday is ok too). The purpose of the update is to keep interested parties (including the VP of Engineering, and peer managers) informed of what is going on in each part of our organization. The format of this update is at the discretion of the sender, but it should be prose that generally answers the following questions:
+
+- What important things happened last week?
+- What are the plans for this week?
+- What issues should my peer-team/manager be aware of?
```

Comment:
I'd like to propose adding the additional question (or similar):

- What if anything can we improve upon in the upcoming week?

Reply:
If it's a thing you _are going to_ improve, then it feels like it should go into "plans for this week". If it's a thing you'd like us to discuss, then it doesn't feel like a status update, and probably belongs in our weekly sync.

Do you have a specific thing in mind that you think should go into a new section?

Reply:
I agree with Chris, and will also add that if something was painful and you want it to improve, expressing that pain would fit into "What should my peer-team/manager be aware of?"

I tried to make it clear in the docs that I do not expect the updates to answer exactly and only these questions, so the list of questions here isn't that important, just a guide.

Reply:
I guess it could fall into the _What issues should my peer-team/manager be aware of?
_.  There's a reality that I always feel that if explicit improvements aren't called out as a target, I feel that they end up being buried in the day-to-day.

Reply:
I do agree with you there. But I'm not sure a status update is the place for it. In my 1:1s, two of my standard questions are "What could have gone better since our last 1:1?" (after asking what went particularly well since our last 1:1), and "What could I be doing more or less of?" The first is oriented toward personal/team improvements for the engineer, and the second for my own improvement.

I'm sure there are other places we could bake this into our routines. I'd like to know what's worked well for you in the past, but maybe we should take this conversation out of Github :-)